package.json

```diff
@@ -35,6 +35,7 @@
     "lodash": "4.6.1",
     "mkdirp": "0.5.1",
     "moment": "2.12.0",
+    "path-is-inside": "1.0.1",
```

Comment:
> We can use path.basename to fix writing outside.

Do you mean that `../../MZuckerberg` silently becomes `MZuckerberg`, or should it do it interactively, like;

```js
if (path.basename(user) !== user) {
    // interactively ask if path.basename(user) should be used instead - else error
}
```